src/mlpack/bindings/go/print_method_init.hpp

```diff
@@ -0,0 +1,170 @@
+/**
+ * @file print_method_init.hpp
+ * @author Yasmine Dumouchel
+ *
+ * Print a config struct initialization function for the optional
+ * parameter of a method for a Go binding.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_BINDINGS_GO_PRINT_METHOD_INIT_HPP
+#define MLPACK_BINDINGS_GO_PRINT_METHOD_INIT_HPP
+
+#include <mlpack/prereqs.hpp>
+#include "get_go_type.hpp"
+#include "strip_type.hpp"
+
+namespace mlpack {
+namespace bindings {
+namespace go {
+
+/**
+ * Print parameter with it's default value for a standard option type.
```

Comment:
Tiny typo that jumped out at me, this should be `its`. Sorry for being pedantic. :)